ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -332,6 +336,8 @@ private void drawRectangularBackgroundWithBorders(Canvas canvas) {
     if (getBorderWidth(Spacing.LEFT) > 0 || getBorderWidth(Spacing.TOP) > 0 ||
         getBorderWidth(Spacing.RIGHT) > 0 || getBorderWidth(Spacing.BOTTOM) > 0) {
 
+      mPaint.setPathEffect(mPathEffectForBorderStyle);
```

Comment:
This relies on someone setting a variable before calling this method. What do you think about passing the `mPathEffectForBorderStyle` as a second argument to this method just for clarity?